.github/workflows/sanity-test.yml

```diff
@@ -315,7 +315,7 @@ jobs:
 
     # Only aarch64 needs this, but it doesn't hurt anything
     - name: Install qemu/docker
-      run: docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+      run: docker run --privileged --rm tonistiigi/binfmt --install all
```

Comment:
```
temporarily downgrade to old ubuntu as 24.04 likes to segfault in the middle of the build
```
Search for this, and remove them, also update the image to latest ubuntu.

Reply:
you can also just install aarch64 only as it does not need anything else